public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -53,6 +53,9 @@ function enqueue_assets() {
 		return;
 	}
 
+	/** Load in admin post functions for `get_default_post_to_edit`. */
+	require_once ABSPATH . 'wp-admin/includes/post.php';
```

Comment:
I'm not sure if this will have any adverse effects, loading this file on the frontend. It hasn't in my testing, but I'd like a gut-check.

Reply:
🤔  

I don't see any logic being executed in it, just functions declared. I can't think of any side effects, unless some logic is added in the future, but that seems unlikely to me 👍🏻